arangod/V8Server/v8-actions.cpp

```diff
@@ -411,7 +411,7 @@ v8::Handle<v8::Object> TRI_RequestCppToV8(v8::Isolate* isolate,
   clientArray->Set(AddressKey, TRI_V8_STD_STRING(isolate, info.clientAddress));
   clientArray->Set(PortKey, v8::Number::New(isolate, info.clientPort));
   TRI_GET_GLOBAL_STRING(IdKey);
-  clientArray->Set(IdKey, TRI_V8_STD_STRING(isolate, taskId));
+  clientArray->Set(IdKey, TRI_V8_STD_STRING(isolate, std::string("0")));
```

Comment:
Why is this always "0" now? It used to be clientTaskId before, wondering if this is obsolete functionality.

Reply:
I am happy to add it if you figure out what it was used before